### PR TITLE
186058 - adding tooltip and question help for Residence

### PR DESCRIPTION
**Pull request by MarcoGoC**
### Description
- Adding Residence Tooltip and QuestionHelp 

#### List of proposed changes:
- see above 

### What to test for/How to test
- Compare test from task to localhost text

### Additional Notes
-

### CODE REVIEW

File: i18n/api/en.ts [hunk 1]
```diff
@@ -169,6 +169,10 @@ const en: Translations = {
       'Only count the number of years since the age of 18.',
     [FieldKey.PARTNER_YEARS_IN_CANADA_SINCE_18]:
       "If you're not sure of the exact number, you may enter an estimate.",
+    [FieldKey.LIVED_ONLY_IN_CANADA]:
+      'Do not include periods when you were outside Canada for at least 6 months at a time. Some exceptions apply, such as working for a Canadian employer abroad.',
+    [FieldKey.LIVED_ONLY_IN_CANADA]:
```
Comment (alex-solo) [inline i18n/api/en.ts]: These are duplicated field keys. I'm guessing the other one is supposed to be for partner

**Review by alex-solo — DISMISSED** [review]
Noticed duplication in en.ts and fr.ts files

File: i18n/api/en.ts (hunk 1 above)
Comment (MarcoGoC) [reply i18n/api/en.ts]: good catch, that's correct

**Review by alex-solo — APPROVED** [review]
looks good!

**Pull request merged**